pyproject.toml

```
@@ -75,10 +75,16 @@ addopts = ["-ra", "--strict-markers", "--strict-config", "--showlocals"]
 log_cli_level = "INFO"
 xfail_strict = true
 
+
+
 [tool.coverage.run]
 source = ["mqt.bench", "mqt.benchviewer"]
 
+
```

```suggestion
[tool.coverage.run]
source = ["mqt.bench", "mqt.benchviewer"]

```
seems to be an unnecessary whitespace change